CHANGELOG.rst

```diff
@@ -24,6 +24,7 @@ in development
 * Fix key name for error message in liveaction result. (bug-fix)
 * Fix 500 API response when rule with no pack info is supplied. (bug-fix)
 * Fix bug in trigger-instance re-emit (extra kwargs passed to manager is now handled). (bug-fix)
+* Fix bug in triggers emitted on key value pair changes and sensor spawn/exit. (bug-fix)
```

Comment:
Might be good to add a bit more context in the second sentence. "...When dispatching those triggers, the reference used didn't contain the pack names which meant it was invalid and lookups in the rules engine would fail." or similar.